<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>34</y>
      <width>1517</width>
      <height>679</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Saida</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Entrada</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo Equip.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modelo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nº chamado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Solicitante</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Patrimônio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Defeito</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Atividade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Solução</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="bvoltar">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>732</y>
      <width>99</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
background-color: rgb(255, 140, 0);
border-color: rgb(255, 154, 152);
border-radius: 5px;
padding: 5px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>546</y>
      <width>69</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bgerarpdf">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>732</y>
      <width>147</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 191, 0);
border: 2px solid black;
border-color: rgb(0, 170, 0);
border-radius: 5px;
padding: 5px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Gerar Relatório</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seditar">
    <property name="geometry">
     <rect>
      <x>782</x>
      <y>732</y>
      <width>99</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
background-color: rgb(85, 0, 127);
border-color: rgb(140, 0, 230);
border-radius: 5px;
padding: 5px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bexcluirsaida">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>732</y>
      <width>99</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-color: rgb(255, 0, 0);
border-radius: 5px;
padding: 5px;
background-color: rgb(255, 55, 55);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>235</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Pesquisar Nº Chamado</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
